graphql/routes.go

```diff
@@ -129,7 +131,22 @@ func routesForSchema(pattern string, execute executeQueryFunc) []types.Route {
 			Method:  http.MethodGet,
 			Pattern: pattern,
 			Handler: http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-				result := execute(r.URL.Query().Get("query"), r.URL.Path, r.Context())
+				vars := r.URL.Query().Get("variables")
+
+				var variables map[string]interface{}
+				if len(vars) > 0 {
+					err := json.Unmarshal([]byte(vars), &variables)
```

Comment:
you might want a test that stresses some of the behaviors of parsing JSON literals into `interface{}` - specifically int and float

... or maybe you already have that handled and there just wasn't a way to pass down these variables before now?

Reply:
Good idea. We already handle this (i think) at a lower level, but couldn't hurt.